### PR TITLE
2FA post

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2022-05-17-requiring-2fa-for-the-ropensci-github-organization/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+After we pulled the switch on April 19th, a few people reached out to tell us they had enabled 2FA.
+It was straightforward to add them back to the organization as GitHub lets you choose to [_re-instate_ an organization member](https://docs.github.com/en/organizations/managing-membership-in-your-organization/reinstating-a-former-member-of-your-organization) within three months of their removal, so they get added to the teams and repos they used to belong to.
+We strove to re-instate people timely.
+
+If you were an organization member or outside collaborator who was removed from the organization, please reach out to us if you have enabled 2FA and would like to be reinstated, or if you have any question about 2FA.
+
+## Conclusion
+
+Two-factor authentication is now required for our main GitHub organization.
+It is a change for the best of most, but that might have created some pain and frustration for a few people depending on when they got the notification.
```
Comment (steffilazerte) [inline content/blog/2022-05-17-requiring-2fa-for-the-ropensci-github-organization/index.md:88]:
```suggestion
It was a change made for the best for most people, but which might have created some pain and frustration for a few people depending on when they got the notification.
```

File: content/blog/2022-05-17-requiring-2fa-for-the-ropensci-github-organization/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+We thank everyone for their collaboration and understanding.
+
+As a further GitHub security step for us and you, dear reader, let's mention
+- pruning repositories from former, now inactive, collaborators;
```
Comment (steffilazerte) [inline content/blog/2022-05-17-requiring-2fa-for-the-ropensci-github-organization/index.md:92]: Link to info on this?

File: content/blog/2022-05-17-requiring-2fa-for-the-ropensci-github-organization/index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+---
+title: Requiring 2FA for our main GitHub Organization
```
Comment (steffilazerte) [inline content/blog/2022-05-17-requiring-2fa-for-the-ropensci-github-organization/index.md:2]:
```suggestion
title: Requiring 2FA for Our Main GitHub Organization
```

File: content/blog/2022-05-17-requiring-2fa-for-the-ropensci-github-organization/index.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  - security
+  - tech notes
+package_version: 0.1.0
+description: How we went about requiring two-factor authentication for our main GitHub organization
+---
+
+Following our recent post on ["Safeguards and Backups for GitHub Organizations"](/blog/2022/03/22/safeguards-and-backups-for-github-organizations/), one month ago we went one step further and made two-factor authentication (2FA) required for all members and outside collaborators of our main organization, ropensci.
+Here is how we went about it.
+
+## Why (not) requiring 2FA?
```
Comment (steffilazerte) [inline content/blog/2022-05-17-requiring-2fa-for-the-ropensci-github-organization/index.md:19]:
```suggestion
## Why (not) require 2FA?
```

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle,

Looks good! I have a couple of wording suggestions, and I wonder if we should use `ropensci` to refer to the organization and distinguish it from rOpenSci? Other than that, thanks for the informative read :) I'm always so amazed by all that I learn. Glad to see that my security practices are up to snuff (whew!). 

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case - Will be ;)
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on [index.md](http://index.md/)
* [X] I ran `roblog::ro_check_urls()` on [index.md](http://index.md/)
* [X] I ran a spell-check on [index.md](http://index.md/)
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

**Comment by maelle** [comment]
@steffilazerte thanks for the thorough review & your kind words! :smile_cat:

**Pull request merged**